key-manager-test/src/test/scala/org/bitcoins/keymanager/util/HdUtilTest.scala

```diff
@@ -35,7 +35,7 @@ class HdUtilTest extends KeyManagerUnitTest {
       HDUtil.getXpubVersion(HDPurposes.NestedSegWit, TestNet3) == NestedSegWitTestNet3Pub)
 
     assertThrows[IllegalArgumentException] {
-      HDUtil.getXpubVersion(HDPurpose(-1), TestNet3)
+      HDUtil.getXpubVersion(HDPurpose(1), TestNet3)
```

Comment:
These we're causing an `IllegalArgumentException` from `BIP32Node` not from `HDUtil`